app/webpacker/components/CompetitionForm/FormSections/Admin.js

```diff
@@ -1,16 +1,23 @@
 import React from 'react';
-import { InputBoolean } from '../../wca/FormBuilder/input/FormInputs';
+import { InputBoolean, InputBooleanSelect, InputNumber } from '../../wca/FormBuilder/input/FormInputs';
+import ConditionalSection from './ConditionalSection';
 import { useStore } from '../../../lib/providers/StoreProvider';
 import SubSection from '../../wca/FormBuilder/SubSection';
+import { useFormObject } from '../../wca/FormBuilder/provider/FormObjectProvider';
 
 export default function Admin() {
+  const { admin } = useFormObject();
   const { isAdminView, isPersisted } = useStore();
   if (!isPersisted || !isAdminView) return null;
 
   return (
     <SubSection section="admin">
       <InputBoolean id="isConfirmed" />
       <InputBoolean id="isVisible" />
+      <InputBooleanSelect id="autoAcceptEnabled" required />
+      <ConditionalSection showIf={admin.autoAcceptEnabled}>
+        <InputNumber id="autoAcceptDisableThreshold" />
+      </ConditionalSection>
```

Comment:
Why do you need to control these through the WCAT Competition Form?
It feels like it would be much easier if the organizers can control these relevant form fields through the registration admin panel directly.